src/plugin/parsers/model_parser.cpp

```diff
@@ -302,7 +311,7 @@ void parseData( MFnMesh & fnmesh, wr::MeshData<wr::Vertex>& mesh_data )
 			}
 			else
 			{
-				assert( false ); //could not get triangles
+				LOGC("Could not get the triangles.");
 				MGlobal::displayInfo( status.errorString() );
```

Comment:
Do we want to `displayInfo` it crashes on `LOGC` anyway..
Either change it to `LOGE`, switch them around, or remove `displayInfo`

Reply:
Done